### DISCUSSION TITLE
Maintainer Handover

### DISCUSSION THREAD

**Discussion by tttroyyy** [NFT-DAO/Governance-HOLON#105] [Gov-HOLON-GitHub]
This post is intended to begin a handover process, as suggested by @stephen-rowan, to bring a Maintainer onboard to administer the NFT-DAO and Governance repos. Stephen, what process might be recommended for this handover to take place? Is it possible to have more than one maintainer?

**Comment by stephen-rowan** [comment]
Hi @tttroyyy - at a minimum all that is needed is a member of NFT-DAO to become the Admin for the Governance-HOLO and Standards repos and handle pull requests etc. I think any Organisation Admin can do this. The GitBook repo is less straightforward as it is on my profile. It would need to be forked to the NFT-DAO organisation and connected to NFT-DAO's GitBook account (which I setup in March/April).

As I said to @severoni1573 earlier I am happy to continue GitHub training to support the continuance of an open-source repo.

S

**Reply by tttroyyy** [reply]
Thank you @stephen-rowan, I think Rich has organised @larry-beck to do this.

I can't make this week, but I am open to learning more. I hope its OK if I ask some questions here and there, now that I'm beginning to navigate the repos more?

Thanks, Troy

**Reply by stephen-rowan** [reply]
That OK Troy - I have begun some handover with @larry-beck on the GitBook. 
Once this repo is handed over either @larry-beck can merge changes or update permissions for other NFT-DAO contributors to do this.

All the Best

S

**Reply by tttroyyy** [reply]
Thank you Stephen!
